cmd/run.go

```diff
@@ -429,6 +432,109 @@ func init() {
 	RootCmd.AddCommand(RunCmd)
 }
 
+func executeRun(runFilePath string, apps []runfileconfig.App) (bool, error) {
+	var exitWithError bool
+	// setup shutdown notify channel.
+	sigCh := make(chan os.Signal, 1)
+	setupShutdownNotify(sigCh)
+
+	runStates := make([]*runExec.RunExec, 0, len(apps))
+	for _, app := range apps {
+		// Get Run Config for different apps.
+		runConfig := app.RunConfig
+
+		// Validate validates the configs and modifies the ports to free ports, appId etc.
+		err := runConfig.Validate()
+		if err != nil {
+			print.FailureStatusEvent(os.Stderr, "Error validating run config for app %s present in %s: %s", runConfig.AppID, runFilePath, err.Error())
+			exitWithError = true
+			break
+		}
+		appLogWriter, err := app.GetAppLogFileWriter()
+		if err != nil {
+			print.FailureStatusEvent(os.Stderr, "Error getting log file for app %s present in %s: %s", runConfig.AppID, runFilePath, err.Error())
+			exitWithError = true
+			break
+		}
+		daprdLogWriter, err := app.GetDaprdLogFileWriter()
+		if err != nil {
+			print.FailureStatusEvent(os.Stderr, "Error getting log file for app %s present in %s: %s", runConfig.AppID, runFilePath, err.Error())
+			exitWithError = true
+			break
+		}
+		runState, err := startDaprdAndAppProcesses(&runConfig, app.AppDirPath, sigCh, daprdLogWriter, daprdLogWriter, appLogWriter, appLogWriter)
+		if err != nil {
+			print.FailureStatusEvent(os.Stdout, "Error starting Dapr and app: %s", err.Error())
+			exitWithError = true
+			break
+		}
+		// Store runState in an array.
+		runStates = append(runStates, runState)
+
+		// Metadata API is only available if app has started listening to port, so wait for app to start before calling metadata API.
+		// The PID is put as 0, so as to not kill CLI process when any one of the apps is stopped.
+		_ = putCLIProcessIDInMeta(runState, 0)
+
+		if runState.AppCMD.Command != nil {
+			_ = putAppCommandInMeta(&runConfig, runState)
+		} else {
+			print.StatusEvent(runState.DaprCMD.OutputWriter, print.LogSuccess, "You're up and running! Dapr logs will appear here.\n")
```

Comment:
`Dapr logs will appear here` - > Are we printing any dapr logs on console ?
if not may be we can use "app_id and daprd up and running"

Reply:
it is printing to the provided stream ... 
it can be console or a file writer ...